modules/spring-config/src/main/java/org/mule/config/spring/parsers/specific/RetryPolicyDefinitionParser.java

```diff
@@ -62,7 +75,7 @@ protected void preProcess(Element element)
         if (StringUtils.isNotEmpty(element.getAttribute("asynchronous")))
```

Comment:
not directly related, but please file a jira about removing this and add the todo